src/service/comparison_vs_peers/comparison_vs_peers_service.py

```diff
@@ -33,14 +33,15 @@ def get_comparison_vs_data(self, data: dict, access: bool) -> list:
             company_revenue = company_data.get("actuals_revenue")
             self.report.calculate_metrics(company_data)
             if not access and id not in allowed_companies:
-                self.report.anonymized_company(company_data, allowed_companies)
+                self.report.anonymize_name(company_data, allowed_companies)
```

Comment:
Why do you have the same validation inside the function and in line 35?  `id not in allowed_companies`

Reply:
I have removed the validation within the function